docs/dsp-tools-xmlupload.md

```diff
@@ -11,7 +11,7 @@ The command to import an XML file on a DSP server is described [here](./dsp-tool
 
 The import file must start with the standard XML header:
 
-```xml
```

Comment:
If it's invalid XML, it's rather a pain that it is marked XML. (PyCharm detects this as a problem)